river/View.zig

```diff
@@ -502,7 +502,7 @@ pub fn map(view: *Self) !void {
         view.pending.ssd = ssd;
     }
 
-    if (server.input_manager.defaultSeat().focused_output) |output| {
+    if (if (server.config.output_rules.match(view)) |output| output else server.input_manager.defaultSeat().focused_output) |output| {
```

Comment:
This will cause a crash should the output be disconnected. Proper fix is to use output name /identifier in the rule instead of `*Output`. Also the line is a it to long.